addons/mail/wizard/mail_compose_message.py

```diff
@@ -700,19 +703,37 @@ def _action_send_mail_mass_mail(self, res_ids, auto_commit=False):
 
         return mails_sudo
 
-    def action_save_as_template(self):
-        """ hit save as template button: current form value will be a new
-            template attached to the current document. """
+    def action_save_as_mail_template(self):
```

Comment:
Does not really save as mail template ... more "open template creation wizard" than creating templates.